ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -377,6 +397,65 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
           {filters}
         </small>
       )}
+    </div>
+  )
+}
+
+GenotypeQuality.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+  showSampleType: PropTypes.bool,
+}
+
+const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById }) => {
+  const individualGenotypes = variant.genotypes[individual.individualGuid]
+  if (!individualGenotypes) {
+    return null
+  }
+  const genotypes = (Array.isArray(individualGenotypes) ? individualGenotypes : [individualGenotypes]).sort(
+    (a, b) => SAMPLE_TYPE_DISPLAY_ORDER.indexOf(a.sampleType) - SAMPLE_TYPE_DISPLAY_ORDER.indexOf(b.sampleType),
+  )
+
+  if (!isCalled(genotypes[0].cn) && !isCalled(genotypes[0].numAlt)) {
+    return <b>NO CALL</b>
+  }
+
+  if (genotypes[0].otherSample) {
```

Comment:
I would add a comment stating that this is to support legacy elasticsearch formatting